bot/cogs/defcon.py

```diff
@@ -115,11 +123,26 @@ def mod_log(self) -> ModLog:
         """Check the DEFCON status or run a subcommand."""
         await ctx.invoke(self.bot.get_command("help"), "defcon")
 
-    async def _defcon_action(self, ctx: Context, action: Awaitable[None], action_type: str) -> None:
-        """Providing a structured way to do an defcon actions - enabled / disabled / updated."""
+    async def _defcon_action(self, ctx: Context, days: int, enabled: bool) -> None:
```

Comment:
Hmm... having second thoughts on the action type inference. I feel like since either way a second parameter (`enabled` boolean at the moment) is needed in addition to `days`, that second parameter may as well just be the enum rather than a boolean to avoid the inference. What do you think?

Reply:
Hmm, I think that's a good idea, it is better to be explicit and pass in an `Action.Enabled` / `Action.Disabled` or `Action.Updated` instead of simple boolean, so an Enum will do wonder here.